doc/lightning_tensor/device.rst

```diff
@@ -0,0 +1,186 @@
+Lightning Tensor device
+=======================
+
+The ``lightning.tensor`` device is a tensor network simulator device. The device is built on top of the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`__ from the NVIDIA cuQuantum SDK, enabling GPU-accelerated simulation of quantum tensor network evolution.
+
+A ``lightning.tensor`` device with the default setup can be simply loaded using:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device("lightning.tensor", wires=100)
+
+By default, the device employs the ``Matrix Product State (MPS)`` tensor network approximation to represent the quantum state. 
+Within this framework, the maximum bond dimension (``max_bond_dim``) defaults to ``128``. To manage computational complexity, 
+singular value truncation is utilized, with the default threshold for truncation (``cutoff``) set at ``0``. Additionally, the 
```

Comment:
```suggestion
Within this framework, the maximum bond dimension (``max_bond_dim``) defaults to ``128`` and the truncation cutoff (``cutoff``) to ``0``. Computational complexity is bounded satisfying either of the criteria, keeping the ``max_bond_dim`` largest singular values, or truncating the singular values smaller than ``cutoff``.
Additionally, the 
```

Reply:
reformated the context above with bullet points